app/models/question.rb

```diff
@@ -5,12 +5,12 @@ class Question < ActiveRecord::Base
 
 
 
-  def post_question question
+  def post_question question 
```

Comment:
Remove the extra spacing after the last character in the lines in this file. If you're using Sublime, you may want to use the setting `trim_trailing_white_space_on_save` (see [documentation](http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/reference/settings.html)).